model/note/import.go

```diff
@@ -67,60 +69,128 @@ func ImportFile(inst *instance.Instance, newdoc, olddoc *vfs.FileDoc, body io.Re
 	return nil
 }
 
-func importReader(inst *instance.Instance, doc *vfs.FileDoc, reader io.Reader, schema *model.Schema) (*model.Node, error) {
+func ImportImages(inst *instance.Instance, olddoc *vfs.FileDoc) error {
+	inst.Logger().WithNamespace("notes").
+		Infof("importing images from note: %s", olddoc.ID())
+	schemaSpecs := DefaultSchemaSpecs()
+	specs := model.SchemaSpecFromJSON(schemaSpecs)
+	schema, err := model.NewSchema(&specs)
+	if err != nil {
+		return fmt.Errorf("failed to read note schema: %w", err)
+	}
+
+	fs := inst.VFS()
+	file, err := fs.OpenFile(olddoc)
+	if err != nil {
+		return fmt.Errorf("failed to open file for note images import: %w", err)
+	}
+
+	content, images, err := importReader(inst, olddoc, file, schema)
+	if err := file.Close(); err != nil {
```

Comment:
[nit] maybe rename `err` in `cerr` to avoid confusion with `err` from the previous line